webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/BreadcrumbSwitcherPopover.js

```diff
@@ -37,9 +38,9 @@ const BreadcrumbSwitcherPopover = ({
       </div>
     );
   } else {
-    const handleItemClick = (item) => {
+    const handleItemClick = (e, item) => {
       onResourceClick(item);
-      if (item.onClick) item.onClick();
+      if (itemOnClick) itemOnClick(e, item.url);
```

Comment:
Btw how is it different from the `onResourceClick` above?

Reply:
The `onResourceClick` callback is for internal use for calling the `closeSwitcher` action.
looks like a rename would clarify the meaning